### DISCUSSION TITLE
Does the Teensy give data to the pi like a game controller?

### DISCUSSION THREAD

**Discussion by reklm-e** [FundamentalFrequency/.github#26] [Q&A] [answered]
Could i hook a keyboard or smth other to the pi and woold it work after configuring or does the teensy give the pi a other signal via usb?

**Comment by stonepreston** [comment, marked as answer]
It sends midi data. If you plugged in a midi keyboard you could play notes, but wouldn’t be able to control anything else with it as the DAW listens for specific midi CC commands

**Reply by reklm-e** [reply]
thank you for your answer is there a specific list for the midi command ( i think they were 3 bit each)

**Reply by stonepreston** [reply]
Look [here](https://github.com/FundamentalFrequency/LMN-3-DAW/blob/main/Source/Modules/app_services/MidiCommandManager/MidiCommandManager.h) starting at line 170 is where the command mappings are defined

**Reply by reklm-e** [reply]
Thank you This helped
